.eslintrc.js

```diff
@@ -22,7 +22,12 @@ module.exports = {
     },
     requireConfigFile: false,
   },
-  ignorePatterns: ['**/tz/**', 'index.stories.jsx', 'index.amp.stories.jsx'],
+  ignorePatterns: [
+    '**/tz/**',
+    'index.stories.jsx',
+    'index.amp.stories.jsx',
+    '.storybook/**/*',
```

Comment:
For `@typescript-eslint/prefer-optional-chain` to work correctly without erroring, I've had to disable linting of anything in .storybook, this is because the [tsconfig](https://github.com/bbc/simorgh/blob/44dc892ef89941c8c5a55ecc76665ebfda96d0c0/tsconfig.json#L44) hasn't been pointed to .storybook, [when I added it](https://github.com/bbc/simorgh/pull/12646/commits/9c4a398d6d92dd62d26bf62b9e418f9a51a47f76) it surfaced  a lot of typescript errors, too many to fix in this PR.

So we either need to a follow up PR to fix the TS errors in .storybook and reinstate linting of the folder, or we don't use this new rule until we do those fixes.

Reply:
This is fine - the components within the storybook directory are mainly for Component Health, and are written in TS. For me, they are much less important than actual components which we use the Simorgh app. Do we even need a follow up ticket to fix this in the future? No harm in creating one, I suppose, but it's low priority.